apps/signing/tests/test_views.py

```diff
@@ -168,6 +172,50 @@ def test_version_exists(self):
         assert response.status_code == 200
         assert 'processed' in response.data
 
+    @mock.patch('devhub.tasks.Version.from_upload')
+    def test_version_exists_with_pk(self, version_from_upload):
+        # Mock Version.from_upload so the version won't be created.
+        self.create_version('3.0')
```

Comment:
The fact that you need a mock here suggests to me that you should be breaking `create_vesion()` down into more of a factory approach:
````
addon = Addon.objects.create(...)
version = ...
file = ...
````
I know that's a lot more work and doesn't need to come in this patch. I think it highlights why relying on HTTP requests for test setup can be awkward (it's also usually a lot slower).